src/compose/ComposeBox.js

```diff
@@ -535,11 +535,12 @@ const ComposeBox: React$AbstractComponent<Props, ImperativeHandle> = forwardRef(
 
     return result;
   }, [
+    messageInputState,
     destinationNarrow,
     mandatoryTopics,
+    isEditing,
     numUploading,
     anyQuoteAndReplyInProgress,
-    messageInputState,
   ]);
```

Comment:
nit: No need to change the order of existing items in this array (`messageInputState`)